proto/pixels.proto

```diff
@@ -81,12 +83,18 @@ enum CompressionKind {
 message Footer {
     // data schema of this file
     repeated Type types = 1;
+    // hidden timestamp column type
+    optional Type hiddenType = 2;
     // file level column statistics
-    repeated ColumnStatistic columnStats = 2;
+    repeated ColumnStatistic columnStats = 3;
+    // file level hidden timestamp column statistics
+    optional ColumnStatistic hiddenColumnStats = 4;
     // row group information
-    repeated RowGroupInformation rowGroupInfos = 3;
+    repeated RowGroupInformation rowGroupInfos = 5;
     // row group statistics
-    repeated RowGroupStatistic rowGroupStats = 4;
+    repeated RowGroupStatistic rowGroupStats = 6;
+    // row group statistics for hidden timestamp columns
```

Comment:
It is better to add the fields for hidden columns at the end of the Footer. Thus, the new file format will be compatible with the old files.